tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -102,9 +102,9 @@ def test_it_uses_placeholder_display_name_for_focused_user_if_api_call_fails(
         }
 
     @pytest.fixture
-    def h_api_client(self, pyramid_config):
-        svc = mock.create_autospec(HAPIClient, instance=True)
-        pyramid_config.register_service(svc, name="h_api_client")
+    def h_api(self, pyramid_config):
+        svc = mock.create_autospec(HAPI, instance=True)
```

Comment:
```suggestion
        svc = mock.create_autospec(HAPI, instance=True, spec_set=True)
```

Not sure why the `spec_set=True` was missing here already but you always want it unless you know you don't